packages/app-desktop/services/spellChecker/SpellCheckerServiceDriverNative.ts

```diff
@@ -54,10 +54,12 @@ export default class SpellCheckerServiceDriverNative extends SpellCheckerService
 		// https://github.com/electron/electron/issues/25228
 		if (effectiveLanguages.length === 0) {
 			this.session().setSpellCheckerLanguages([]);
+			this.session().setSpellCheckerEnabled(false);
 			return;
 		}
 
 		this.session().setSpellCheckerLanguages(effectiveLanguages);
+		this.session().setSpellCheckerEnabled(true);
```

Comment:
Isn't it the same as this?

```
this.session().setSpellCheckerLanguages(effectiveLanguages);
this.session().setSpellCheckerEnabled(effectiveLanguages.length > 0);
```

Reply:
yes, The changes i have done and 
`this.session().setSpellCheckerLanguages(effectiveLanguages);
this.session().setSpellCheckerEnabled(effectiveLanguages.length > 0);`
are same. 
Should i change my commit to this?

Reply:
yes